examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -37,6 +37,25 @@ void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aC
         "Default DispatchSingleClusterCommand is called, this should be replaced by actual dispatched for cluster commands");
 }
 
+CHIP_ERROR WaitForSessionSetup(chip::Controller::Device * device)
+{
+    constexpr time_t kWaitPerIteration = 1;
```

Comment:
kWaitPerIterationSec seems clearer (add sec  or millis and similar as suffixes to avoid misuse)